mlflow/ml-package-versions.yml

```diff
@@ -413,5 +413,8 @@ diviner:
   models:
     minimum: "0.1.0"
     maximum: "0.1.0"
+    requirements:
+      # https://github.com/alan-turing-institute/sktime/issues/2898
+      ">= 0.0": ["cmdstanpy<1.0.2"]
```

Comment:
@harupy Looks like `cmdstanpy` 1.0.3 was just released. Does it fix the problem? If so, can we use `cmdstanpy!=1.0.2` here?

Reply:
Triggered diviner tests:
https://github.com/mlflow/mlflow/runs/7122436126?check_suite_focus=true

Reply:
They passed.